src/plot/ROOTMultiPlot.cpp

```diff
@@ -77,7 +77,10 @@ ROOTMultiPlot::Construct(){
   for(size_t i = 0; i < fNames.size(); i++){
     fLegend.AddEntry(&fHists.at(i), fNames.at(i).c_str());
     fHists[i].SetLineColor(fColorScheme[i %fNcolors]);
-    fHists.at(i).Draw("same");
+    if(i!=0)
```

Comment:
I tested this in the last week and it did what was expected for just adding a bunch of pdfs and then drawing. But probably wouldnt have if I added more stuff on top of that and drew again